crates/infra/utils/src/codegen/read_write.rs

```diff
@@ -27,6 +28,16 @@ impl CodegenReadWrite {
             let templates_glob = input_dir.join("**/*.tera");
             let mut tera = Tera::new(templates_glob.unwrap_str())?;
 
+            fn snake_case(
+                value: &tera::Value,
+                _params: &std::collections::HashMap<String, tera::Value>,
```

Comment:
nit: Probably worth importing HashMap? This looks like an auto-import using rust-analyzer